src/scopes.ts

```diff
@@ -986,6 +987,13 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.PreferenceManagement],
   },
+  [ScopeName.LLMLogTransfer]: {
+    title: 'LLM Log Transfer',
+    dependencies: [ScopeName.ManageDataInventory],
+    description: 'Ability to transfer logs from LLM to Transcend',
+    type: ScopeType.Modify,
+    products: [TranscendProduct.StructuredDiscovery],
+  }
```

Comment:
Not sure what we had in mind here, but it might make sense to just have a single, general purpose logging API key? Either way, this is good!

Reply:
i think we should do it like this for now. 

we don't want to add an api key to sombra, if you check height ticket,  ideal future state is some sort of reverse proxy where we would remove this entirely